src/PopEngine/index.ts

```diff
@@ -100,6 +110,11 @@ export class PopEngine {
       let container = <Element>document.querySelector('div[pop-id="' + groupId + '"]');
 
       if (isAlreadyShowing && !!container) {
+        if (!!oldPop && !!oldPop.childPops.length) {
```

Comment:
Can we put this into its own function? Or at least leave a comment why this code exists.